react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx

```diff
@@ -251,6 +251,66 @@ MapLayer3dPng.parameters = {
     },
 };
 
+const wellsLayerNoDepthTest = {
```

Comment:
Should we put this story in `DeckGLMap.stories.tsx` since it's a higher level concept?